src/components/FoodPairingBlock/FoodPairingBlock.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './foodPairingBlock.scss';
+
+export default class FoodPairingBlock extends React.PureComponent {
+    static propTypes={
+        foodPairingList: PropTypes.array.isRequired
+    }
+
+    renderListItems=(foodPairings) => {
```

Comment:
Try to think about something to replace `...ListItems`

Reply:
Also, I think you can just create a variable inside `render` method and don't create a separate function. But it's up to you